packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -392,6 +397,14 @@ export default {
                 (!this.address || !this.address.line1);
         },
 
+        /* If phone number is missing both from chckout api and from
+        * `state.AuthToken`, then retrieve the phone number from customer api
+        * This can happen for newly created guest */
+        shouldLoadCustomer () {
+            return this.isLoggedIn &&
```

Comment:
Super nit, but I would keep this in the same line.

Reply:
Will open separate PR